### PR TITLE
Shortened and polished gene expression section

### PR DESCRIPTION
**Pull request by bdo311**
My general feeling is that the sections within "Study" all have a lot of overlapping content (since they were written by different contributors) so I'm trying to remove redundancy.

### CODE REVIEW

**Comment by agitter** [comment]
@bdo311 Thank you very much for editing the Study sections and decoupling the pull requests for the individual topics.  I'll work on reviewing these.

File: sections/04_study.md [hunk 1]
```diff
@@ -30,78 +30,67 @@ and why it has received less attention, @tag:Chen2015_trans_species as example`
...
+tasks. For example, a deep neural network with three hidden layers outperformed
+linear regression in inferring the expression of over 20,000 target genes based
+on a representative, well-connected set of about 1,000 landmark genes
+[@tag:Chen2016_gene_expr]. However, while the deep learning model outperformed
+existing algorithms in nearly every scenario, the model still displayed poor
+performance. The paper was also limited by computational bottlenecks that
+required data to be split randomly into two distinct models and trained
+separately. It is unclear how much performance would have increased if not for
+computational restrictions. Alternatively, epigenetic data may have sufficient
+explanatory power for inference of gene expression: for instance, a
```
Comment (agitter) [inline sections/04_study.md]: Could start a new sentence at "For instance,"

File: sections/04_study.md [hunk 2]
```diff
@@ -30,78 +30,67 @@ and why it has received less attention, @tag:Chen2015_trans_species as example`
...
-being developed to address novel problems such as adjusting for batch effects in
-single-cell RNA-seq data [@tag:Shaham2016_batch_effects]. Moreover, deep
-learning is already well established in the image processing community, so the
-marriage of fluorescence based imaging techniques and deep learning is natural.
-These technologies are growing in popularity and will provide increasingly novel
-perspectives with respect to how cellular heterogeneity impacts gene expression
-coordination within a sample. In general, as the flow of gene expression data
+DeepChrome, [@tag:Singh2016_deepchrome] was shown to improve prediction accuracy
+over existing methods.  Deep learning can also be useful for integrating
+different data types: Liang et al. combine RBMs to integrate gene expression,
```
Comment (agitter) [inline sections/04_study.md]: I also think it works better to start a new sentence at "Liang et al.".

File: sections/04_study.md [hunk 3]
```diff
@@ -30,78 +30,67 @@ and why it has received less attention, @tag:Chen2015_trans_species as example`
...
-are good matches for deep learning. Concurrently, deep learning methods are
-being developed to address novel problems such as adjusting for batch effects in
-single-cell RNA-seq data [@tag:Shaham2016_batch_effects]. Moreover, deep
-learning is already well established in the image processing community, so the
-marriage of fluorescence based imaging techniques and deep learning is natural.
-These technologies are growing in popularity and will provide increasingly novel
-perspectives with respect to how cellular heterogeneity impacts gene expression
-coordination within a sample. In general, as the flow of gene expression data
+DeepChrome, [@tag:Singh2016_deepchrome] was shown to improve prediction accuracy
+over existing methods.  Deep learning can also be useful for integrating
```
Comment (agitter) [inline sections/04_study.md]: Was the phrase "high or low" important?  Did this indicate that they predicted binary expression?

If so, we could reincorporate it by saying "prediction accuracy of binary expression over..." or "of high and low expression over..." to indicate that this is a simpler setting that continuous expression.

File: sections/04_study.md [hunk 4]
```diff
@@ -30,78 +30,67 @@ and why it has received less attention, @tag:Chen2015_trans_species as example`
...
-
-Deep learning applied to gene expression data is in its infancy but the future
-is bright. Many hypotheses can now be interrogated because of increasing
-amounts of data and new developing technologies. For example, there is a
-growing appreciation for the large impact of disease heterogeneity on research
-and treatment strategies for diseases. New technologies are being developed,
-such as single cell RNA-seq and high throughput fluorescence based imaging that
-are good matches for deep learning. Concurrently, deep learning methods are
-being developed to address novel problems such as adjusting for batch effects in
-single-cell RNA-seq data [@tag:Shaham2016_batch_effects]. Moreover, deep
```
Comment (agitter) [inline sections/04_study.md]: Is the `Shaham2016_batch_effects` reference being dropped?  Does it belong in the single cell section?

File: sections/04_study.md [hunk 5]
```diff
@@ -30,78 +30,67 @@ and why it has received less attention, @tag:Chen2015_trans_species as example`
...
+variables, thus ablating many complex gene expression signatures present in
+intermediate and relative numbers.
+
+Deep learning applied to gene expression data is still in its infancy, but the
+future is bright. Many previously untestable hypotheses can now be interrogated
+as deep learning enables analysis of increasing amounts of data generated by new
+technologies. For example, the effects of cellular heterogeneity on basic
+biology and disease etiology can now be explored by single-cell RNA-seq and
+high-throughput fluorescence-based imaging, techniques that will benefit
+immensely from deep learning approaches. As the flow of gene expression data
```
Comment (agitter) [inline sections/04_study.md]: If you want to shorten this even further, we could drop the last sentence "As the flow...".  It's up to you.

File: sections/04_study.md (hunk 3 above)
Comment (bdo311) [inline sections/04_study.md]: Yep the output was binary. Thanks for the catch!

File: sections/04_study.md (hunk 4 above)
Comment (bdo311) [inline sections/04_study.md]: I can certainly add it to the single cell section -- definitely would fit well there.

Comment (bdo311) [inline sections/04_study.md]: #442

**Pull request merged**